rolling-update-kubernetes/src/main/scala/akka/rollingupdate/kubernetes/KubernetesApiImpl.scala

```diff
@@ -296,7 +296,7 @@ PUTs must contain resourceVersions. Response:
     Unmarshal(response.entity)
       .to[String]
       .map(body =>
-        throw new PodCostException(
+        throw new UnauthorizedException(
```

Comment:
doesn't matter what exception is used here, but since `handleUnauthorized` is also used for the revision